test/system/255-auto-update.bats

```diff
@@ -0,0 +1,243 @@
+    fi
+
+    run systemctl status container-$cname
+    if [ $status -ne 0 ]; then
+        die "Non-zero status of systemd unit $SERVICE_NAME, output: $output"
+    fi
+}
+
+# This test can fail in dev. environment because of SELinux.
+# quick fix: chcon -t container_runtime_exec_t ./bin/podman
+@test "podman auto-update - label io.containers.autoupdate=image" {
+    run_podman images
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top image
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman auto-update
+    is "$output" "Trying to pull.*" "Image is updated."
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    [[ "$output" != "$ori_image" ]]
+}
+
+@test "podman auto-update - label io.containers.autoupdate=disabled" {
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top disabled
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman auto-update
+    is "$output" "" "Image is not updated with disabled."
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    is "$output" "$ori_image" "Image hash should not changed."
+}
+
+@test "podman auto-update - label io.containers.autoupdate=fakevalue" {
+    fakevalue=$(random_string)
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top $fakevalue
+    cname=${lines[0]}
+    echo $output
+    run_podman inspect --format "{{.Image}}" $cname
+    ori_image=$output
+
+    sleep 2
+    run_podman ps -a
+    run_podman ? auto-update
+    is "$output" ".*invalid auto-update policy.*" "invalid policy setup"
+    run_podman ps -a
+    sleep 2
+    run_podman inspect --format "{{.Image}}" $cname
+    is "$output" "$ori_image" "Image hash should not changed."
+}
+
+@test "podman auto-update with multiple services" {
+    fakevalue=$(random_string)
+    run_podman inspect --format "{{.Id}}" $IMAGE
+    img_id="$output"
+    cnames=()
+    cnames_update=()
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top image
+    cname_update="${lines[0]}"
+    cnames+=($cname_update)
+    cnames_update+=($cname_update)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine_nginx:latest top image
+    cname_update_1="${lines[0]}"
+    cnames+=($cname_update_1)
+    cnames_update+=($cname_update_1)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top
+    cname_default="${lines[0]}"
+    cnames+=($cname_default)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top disabled
+    cname_disabled="${lines[0]}"
+    cnames+=($cname_disabled)
+    echo $output
+
+    run generate_service $IMAGE quay.io/libpod/alpine:latest top ""
```

Comment:
This is identical to line 139. Is it really necessary?

Reply:
Yes. I want to test if the command can handle different value for this label, and empty value is one of them.